src/core/classes/marketplace-direct.ts

```diff
@@ -274,6 +294,21 @@ export class MarketplaceDirect {
     };
   }
 
+  /**
+   * Accept an offer on a direct listing
+   *
+   * @remarks Accept an offer on a direct listing
+   *
+   * @example
+   * ```javascript
+   * // The listing ID of the asset you want to bid on
+   * const listingId = 0;
+   * // The price you are willing to bid for a single token of the listing
+   * const offeror = "0x...";
```

Comment:
the description and the variable don't line up here

Reply:
wdym? they are both 1 space from the *

Reply:
@jarrodwatts no like... read the description, and then look at what the variable that you're declaring is